app/main/helpers/validation.py

```diff
@@ -282,6 +282,37 @@ def errors(self):
         return errors_map
 
 
+class DOS5Validator(SharedValidator):
+    """Following an accessibility review, a number of questions and answers were changed for DOS 5"""
+    email_validation_fields = {"contactEmailContractNotice", "contactEmail"}
+    percentage_fields = ["subcontractingInvoicesPaid"]
+
+    def is_valid_percentage(self, field):
+        value = self.answers.get(field)
+
+        # Design System guidance is that we should allow users to provide answers with or without units.
+        if isinstance(value, str):
+            value = value.rstrip('%')
```

Comment:
Neat trick! We might need to save this back to `answers`, as otherwise the declaration gets updated with the percentage sign, which is probably not what we want.

![50%%](https://user-images.githubusercontent.com/503614/94800867-1f236080-03dd-11eb-80eb-96007e7b9567.png)

Reply:
Hmm. This makes me quite uneasy. At the moment, the validators do not change any of the parameters they're passed. Can we easily do this in a thread-safe way?

Is it right to modify the user input, or should we be making this change where we display it?

We might want to discuss with you and Brett. As such, we're not going to fix this in this PR.